itsUk45/Property-Pro-Lite-
# #167270354 resolve delete route build error

What does this PR do?

   - resolve bug in the delete route

Description of Task to be completed

    Before a user performs a delete operation, a test case is executed.

How should this be manually tested?
   - clone the repo and switch to the branch bug-develop-api-test-moocha
    -run nodemon app to start the server
    -run npm run coveralls to see the test output
   

Any background context you want to provide?
   -N/A

What are the relevant pivotal tracker stories?
- #2354662
Screenshots (if appropriate)
  -N/A
